dali/test/python/test_torch_pipeline_rnnt.py

```diff
@@ -112,8 +112,7 @@ def normalize_batch(x, seq_len, normalize_type):
             return x
 
     def get_seq_len(self, seq_len):
-        return torch.ceil(seq_len.to(dtype=torch.float) / self.hop_length).to(
-            dtype=torch.int)
+        return seq_len.to(dtype=torch.int) // self.hop_length + 1
```

Comment:
This is a bug in the reference.

Reply:
I don't understand. The new code is a bug or the old one? This one looks broken.

Reply:
The old one is the bug. The new one is the actual length of the spectrogram.